server/lib/nl/detection/context.py

```diff
@@ -217,8 +222,9 @@ def _detect_places(uttr: nl_uttr.Utterance,
       if _handle_answer_places(uttr, child_type, places, cmp_places):
         return places, cmp_places
     else:
-      # There are NO places, so if there are answer places, bail.
-      if _handle_answer_places(uttr, child_type, places, cmp_places):
+      # There are NO places, so if there are answer places or entities, bail.
+      if _handle_answer_places(uttr, child_type, places,
```

Comment:
update comment on why the change?  I don't follow

Reply:
The change is that if entities were detected, we skip getting place from context, which the comment was updated with "... or entities, bail"